test/integration/basic-server.spec.js

```diff
@@ -27,56 +27,56 @@ const Static = Dispatcher.Static
 let server = null
 
 
-  let getData = function(data,timeout) {
-    return new Promise(function(resolve, reject) {
-      if (!timeout) {
+let getData = function(data, timeout) {
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).